src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -60,6 +65,16 @@ class ProductConfigFormController {
     this.initItemConfig()
     this.loadData()
     this.waitForFormInitialization()
+
+    this.$rootScope.$on(brandedCoverFeeCheckedEvent, () => {
```

Comment:
Why does this need an event? How did the regular checkout cover fee do it?

Reply:
They both use events. Regular checkout uses the `cartUpdatedEvent`.

Reply:
The major difference is that in branded checkout, the product config form is always showing on the page; whereas in regular checkout, it is brought up when the user goes to edit a gift (which calls `$onInit()`)